lib/src/main/scala/spinal/lib/Stream.scala

```diff
@@ -534,6 +535,16 @@ class Stream[T <: Data](val payloadType :  HardType[T]) extends Bundle with IMas
     this
   }
 
+  def withCovers(back2BackCycles: Int = 1): this.type  = {
```

Comment:
i would say, back2BackCycles should be 0 by default, as in some design, the hardware will never reach maximal throughput. Ex :  Stream.halfPipe